packages/eslint-plugin-shopify/lib/rules/jquery-dollar-sign-reference.js

```diff
@@ -0,0 +1,152 @@
+var JQUERY_IDENTIFIERS = ['$', 'jQuery'];
+
+var nonChainingMethods = {
+  context: true,
+  get: true,
+  hasClass: true,
+  height: true,
+  index: true,
+  innerHeight: true,
+  innerWidth: true,
+  is: true,
+  offset: true,
+  outerHeight: true,
+  outerWidth: true,
+  position: true,
+  promise: true,
+  scrollLeft: true,
+  scrollTop: true,
+  serialize: true,
+  serializeArray: true,
+  size: true,
+  text: true,
+  toArray: true,
+  triggerHandler: true,
+  val: true,
+  width: true,
+
+  html: function(node) {
```

Comment:
I guess this block of functions should be in alphabetical key order?  To be consistent with the above block :)